### ISSUE TITLE
Extreme Value Theorem, Gumbel distribution and Gumbel Softmax

### ISSUE THREAD

**Issue by vinsis** [mukappalambda/readings#9]
- Some key ideas and history behind the EVT, especially focused on Gumbel
- Application of Gumbel softmax to backpropagate through discrete categorical sampling
- Extension of ideas behind Gumbel softmax to make sampling from a Poisson distrubution differentiable

**Comment by mukappalambda** [comment]
- The Fisher–Tippett–Gnedenko theorem
- [Jang, E., Gu, S., & Poole, B. (2016). Categorical reparameterization with gumbel-softmax. arXiv preprint arXiv:1611.01144.](https://openreview.net/pdf?id=rkE3y85ee)
